retrievalmarket/rtvllog/db.go

```diff
@@ -227,7 +227,8 @@ func (d *RetrievalLogDB) list(ctx context.Context, offset int, limit int, where
 
 		dealState.PayloadCID, err = cid.Parse(payloadCid.String)
 		if err != nil {
-			return nil, fmt.Errorf("parsing payload cid '%s': %w", payloadCid.String, err)
+			//return nil, fmt.Errorf("parsing payload cid '%s': %w", payloadCid.String, err)
+			dealState.PayloadCID = cid.Undef
```

Comment:
Could you explain this change?

Reply:
If we handle the error by returning then UI fails. We get an error saying "parsing payload failed" with whole page going blank. Thus an empty value.

Reply:
Ok makes sense - let's just remove the commented out line then